setup.cfg

```diff
@@ -1,5 +1,66 @@
+[metadata]
+name = pyroma
+version = 3.3.dev0
+description = Test your project's packaging friendliness
+long_description = file: README.rst, CHANGES.txt
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: PyPy
+keywords = pypi, quality, testing
+author = Lennart Regebro
+author_email = regebro@gmail.com
+url = https://github.com/regebro/pyroma
+license = MIT
+project_urls =
+    Source Code = https://github.com/regebro/pyroma
+
+[options]
+zip_safe = True
+include_package_data = True
+packages = find:
+package_dir =
+    = .
+test_suite = pyroma.tests
```

Comment:
(Sorry, thought I submitted this comment a long time ago)

Just FYI, this (and the `test` command it supports) has been deprecated for a number of years (pypa/setuptools#1684) and is planned to be removed (pypa/setuptools#931), so it should be removed here too.

Reply:
Oh, right, forgot about that. I was thinking about actually adding a Pyroma warning for that as well.

Reply:
Good point, though to note, unlike almost all the other things you current check that are standardized package metadata, that's part of the build backend's options rather than the package metadata, specific to one particular build backend and requires reading said backend's bespoke configuration (in one of several places, not all of which are statically introspectable), so you'll want to consider whether it is in scope for Pyroma and worth the effort to implement and maintain, considering unlike Pyroma's other, more abstract and "opinionated" feedback that form the core of its unique value, those warnings/removals are already handled by the backend in question.